res_users_password_security/__openerp__.py

```diff
@@ -5,7 +5,7 @@
 
     'name': 'Password Security',
     "summary": "Allow admin to set password security requirements.",
-    'version': '9.0.1.0.1',
+    'version': '9.0.1.0.2',
```

Comment:
FYI I have to leave this version number, this is already in the wild under our repos. We are migrating here and will be having it removed from the app store for a hopefully seamless transition.